src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -46,7 +46,8 @@ public function render(array $attributes = [], array $separator = []): string
     }
 
     /**
-     * {@inheritDoc}
+     * @inheritDoc
```

Comment:
thats not valid together. see my comment.

Reply:
Please fix the invalid doc block here as per my review, I mentioned this several times before.